x-pack/legacy/plugins/maps/public/layers/tooltips/tooltip_property.ts

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import _ from 'lodash';
+
+export interface ITooltipProperty {
+  getPropertyKey(): string;
+  getPropertyName(): string;
+  getHtmlDisplayValue(): string;
+  getRawValue(): string | undefined;
+  isFilterable(): boolean;
+  getESFilters(): Promise<unknown[]>;
```

Comment:
I'd type this more strongly, to `Promise<PhraseFilter[]>` https://github.com/elastic/kibana/blob/dfcf29afd39506bc78233b555b00eca97c9f4354/src/plugins/data/common/es_query/filters/phrase_filter.ts#L32

`import { PhraseFilter } from '../../../../../../../src/plugins/data/public'`